File: DistanciaPuntosC/Resultado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DistanciaPuntosC
{
    class Resultado : Puntos
    { 
        
        public double Distancia()
        {
            double resultado;
            resultado = ((Math.Pow(Xb - Xa, 2)) + (Math.Pow(Yb - Ya, 2)));
            resultado = (Math.Sqrt(resultado));
            return resultado;
        }
    }

}

File: DistanciaPuntos/Resultado.vb
Public Class Resultado
    Inherits Puntos
    Public Function Distancia() As Double
        Dim resultado As Double
        resultado = ((Math.Pow(XB - XA, 2)) + (Math.Pow(YB - YA, 2)))
        resultado = (Math.Sqrt(resultado))
        Return resultado
    End Function
End Class
